src/factory.css

```diff
@@ -607,3 +607,53 @@ td.taboff:hover {
 .shadowBlock>.blocklyPathDark {
   display: none;
 }
+
+ /* New block popup */
+ 
+#popup {
+  display: none;
+  height: 100%;
+  left: 0;
+  position: absolute;
+  top: 0;
+  width: 100%;
+  z-index: 99999999999999999999999999999;
```

Comment:
This might be excessive.  I assume the issue is Blockly's z-index.  Can you find the specific value in there that you need to be higher then?  Also, add a comment describing why this unusual style is necessary, so the next dev will not remove it (or will understand the implications of doing so).